<?xml version="1.0" ?>
<launch>


  <group ns="human_detection">
    <param name="topic_external_wrench" value="/admittance_control/transformed/external_wrench"/>
    <param name="topic_admittance_ratio" value="/admittance_control/admittance_ratio" />
    <param name="topic_tank_state" value="/admittance/tank_state"/>
    <param name="topic_admittance_ratio" value="/admittance_control/admittance_ratio"/>
    <rosparam command="load" file="$(find ds_admittance_control)/config/admittance_param.yml"/>   
    <node name="human_detection" pkg="ds_admittance_control" type="human_detection_node" output="screen"/>
  </group>

  <node name="reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" output="screen" />





</launch>